sdk/daml_finance_dep.bzl

```diff
@@ -2,6 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
 quickstart = {
-    "version": "576adbc8a51e75f1c9faf1217478ce96b193d735",
-    "sha256": "582a4cc40a1c94178fd3bdbda2af681b584725f5481fa6079749ee9bfe64de06",
+    "version": "00a9d74967831a33407077443a2d8abe2d0cae5a",
```

Comment:
Could you remind me how you derive this version identifier?

Reply:
Item 4.iiii here: https://github.com/digital-asset/daml-finance/blob/main/docs/code-samples/README.md#:~:text=In%20the%20daml%20repository%2C%20update,the%20filename%20which%20was%20hashed).

Reply:
To summarize
- commit got squashed but Github still maintains a reference to it
- we are switching to current `main` for clarity